src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,16 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.CarDaoImpl;
+import mate.jdbc.dao.DriverDaoImpl;
+import mate.jdbc.model.Car;
+import mate.jdbc.service.CarServic;
+import mate.jdbc.service.CarServiceImpl;
+
 public class Main {
     public static void main(String[] args) {
```

Comment:
Test your solution in the main method.
create car
add two drivers
print car
delete the first one from the car
print car again

Reply:
not fixed